src/app/routes/enrolment/models/enrolment-claim.ts

```diff
@@ -92,6 +94,10 @@ export class EnrolmentClaim
     );
   }
 
+  get verifiableCredential(): VerifiableCredential<ICredentialSubject> {
```

Comment:
There is already this check in EnrolmentClaimAbstract under `credential`. Unfortunately interfaces are different, because it returns `VerifiableCredential<RoleCredentialSubject>`. However when I changed `credential` to return `VerifiableCredential<ICredentialSubject>` it works fine. So maybe we could merge those two getters?

Reply:
@dawidgil you are totally right, I got rid of this entirely and just used credential. Great.